### PR TITLE
Update nushell.txt to fit nushell

### PR DESCRIPTION
**Pull request by hackers267**
Change the `def-env` to `def --env` to fit the nushell v8.8, because the `def-env` is deprecated and be removed in v8.8

Fixes https://github.com/ajeetdsouza/zoxide/issues/630

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@
   <div>
     <img src="contrib/warp.png" width="230" alt="Warp" />
   </div>
-  <b>Warp is a modern, Rust-based terminal with AI built in so you and your team can build great software, faster.</b>
+  <b>Warp is a modern, Rust-based terminal with AI built in so you
+    and your team can build great software, faster.</b>
```
Comment (marcelarie) [inline README.md:19]: This is not releated to the PR, right?

Comment (hackers267) [reply README.md:19]: Yes, modify this just because some error in CI

**Comment by ajeetdsouza** [comment]
Thanks!

**Comment by happysalada** [comment]
is there a chance there can be a new release of zoxide for this before mid december ?
Just asking since this will become a hard deprecation in nushell around that time (most likely).
We can always work around this of course if you are busy.
Thank you for making zoxide!

**Comment by happysalada** [comment]
Hey no worries at all !
Thanks again for your work on this !

**Comment by happysalada** [comment]
@ajeetdsouza small bump, the breaking release of nu is about 2 weeks away.

**Comment by happysalada** [comment]
@ajeetdsouza just a friendly ping in case you have a bit of time this weekend.
I'll probably ping again next week if that's alright with you!

**Comment by happysalada** [comment]
@ajeetdsouza another friendly ping. The release containing the breaking change in nushell has just been made. 
No worries at all if you have work, i dont plan on upgrading my system too soon personally, but when you have a moment, it would be nice.

**Comment by poliorcetics** [comment]
As a workaround for people ending up here:

When you get an error about `def-env` not working, replace it by `def --env`:

`zoxide init nushell | str replace "def-env" "def --env" | save -f ...` that way it works automatically and will continue to work on the next zoxide update :)

**Pull request merged**